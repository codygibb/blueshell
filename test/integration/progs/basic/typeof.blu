print typeof(2)

print typeof(23.13)

print typeof(true)

print typeof("hello")

print typeof([1, 2, 3])

print typeof({"key": "val"})

print typeof(func(x) { x + 1 })

print typeof([1, 2, 3].push)

print typeof(typeof)

print typeof(nil)

var i
print typeof(i)

print typeof(func(){}())
