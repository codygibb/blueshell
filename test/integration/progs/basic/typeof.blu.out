int
float
bool
str
list
dict
func
func
func
nil
nil
nil
